debugger.md

```diff
@@ -26,17 +26,17 @@ Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/
 2. Install Dotnet CLI
  * Install Dotnet CLI following the instructions here:  http://dotnet.github.io/getting-started  
  * **Hint for Mac**: Dotnet CLI requires openSSL to work. It will ask you to install this during installation. Don't forget this!
-3. C# Extension for VS Code
- * Get the .vsix file from here: \\\\vsdbgqa\x-plat\vscode-extension\csharp
+3. Install C# Extension for VS Code
+ * Get the .vsix file from here: \\\\vsdbgqa\x-plat\vscode-extension\osx\csharp\csharp-0.3.5.vsix . 
```

Comment:
Let's put the osx one under \\vsdbgqa\x-plat\vscode-extension and not bother telling people of osx in the path etc.